client/apps/user-portal/src/app/sermons/sermon-view/sermon-view.component.ts

```diff
@@ -1,15 +1,58 @@
 import { Component, OnInit } from '@angular/core';
```

Comment:
Just a little nit-picky thing. Don't have to update to merge, but try separating imports like so:

```typescript
// angular imports
import { Component, OnInit } from '@angular/core';

// third-party imports
import { Observable } from 'rxjs';

// local imports
import { Model } from '@localapp-model';
```